[Onewire] PR #4140 broke the build

<s>It appears the changes in PR #4140 do not build successfully, and I didn't catch it before merging them.</s>

Further review shows my manual merge caused this.

Fix coming shortly.

````
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 7.711 s
[INFO] Finished at: 2017-01-12T01:57:19-05:00
[INFO] Final Memory: 36M/261M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.eclipse.tycho:tycho-compiler-plugin:0.24.0:co
mpile (default-compile) on project org.openhab.binding.onewire: Compilation fail
ure: Compilation failure:
[ERROR] openhab\bundles\binding\org.openhab.binding.onewire\src\main\java\org\openhab\binding\onewire\internal\OneWireBinding.java:[132]
[ERROR] String lvStringValue = lvWritableBindingConfig.convertTypeToString(pvCom
mand);
[ERROR] ^^^^^^^^^^^^^^^^^^^
[ERROR] The method convertTypeToString(Type) from the type AbstractOneWireDevice
PropertyWritableBindingConfig refers to the missing type Type
[...]
[ERROR] openhab\bundles\binding\org.openhab.binding.onewire\src\main\java\org\openhab\binding\onewire\internal\deviceproperties\AbstractOneWireDevicePropertyWritableBindingConfig.java:[24]
[ERROR] private static final Logger logger = LoggerFactory
[ERROR] ^^^^^^
[ERROR] Logger cannot be resolved to a type
[...]
[ERROR] openhab\bundles\binding\org.openhab.binding.onewire\src\main\java\org\openhab\binding\onewire\internal\deviceproperties\AbstractOneWireDevicePropertyWritableBindingConfig.java:[55]
[ERROR] abstract protected String convertTypeToUnmodifiedString(Type pvType);
[ERROR] ^^^^
[ERROR] Type cannot be resolved to a type
[...]
[ERROR] 28 problems (28 errors)
````

--- a/bundles/binding/org.openhab.binding.onewire/src/main/java/org/openhab/binding/onewire/internal/deviceproperties/AbstractOneWireDevicePropertyBindingConfig.java
+++ b/bundles/binding/org.openhab.binding.onewire/src/main/java/org/openhab/binding/onewire/internal/deviceproperties/AbstractOneWireDevicePropertyBindingConfig.java
@@ -12,7 +12,9 @@ import java.util.ArrayList;
 
 import org.openhab.binding.onewire.internal.OneWireBindingConfig;
 import org.openhab.binding.onewire.internal.deviceproperties.modifier.OneWireTypeModifier;
+import org.openhab.core.types.Type;
 import org.openhab.model.item.binding.BindingConfigParseException;
+import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 /**

--- a/bundles/binding/org.openhab.binding.onewire/src/main/java/org/openhab/binding/onewire/internal/deviceproperties/AbstractOneWireDevicePropertyWritableBindingConfig.java
+++ b/bundles/binding/org.openhab.binding.onewire/src/main/java/org/openhab/binding/onewire/internal/deviceproperties/AbstractOneWireDevicePropertyWritableBindingConfig.java
@@ -9,7 +9,9 @@
 package org.openhab.binding.onewire.internal.deviceproperties;
 
 import org.openhab.binding.onewire.internal.deviceproperties.modifier.OneWireTypeModifier;
+import org.openhab.core.types.Type;
 import org.openhab.model.item.binding.BindingConfigParseException;
+import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 /**

